src/status_im/subs.cljs

```diff
@@ -715,6 +715,16 @@
  (fn [{:keys [unviewed-messages-count]}]
    unviewed-messages-count))
 
+(re-frame/reg-sub
+ :chats/last-message-content
```

Comment:
why do we need this subscription?

Reply:
I followed what was done for `unviewed-messages-count` that was done already for `undread-count` badge in Chat list.

For the problem of unread message (displayed as subtitle of Chat list) not updating, that @churik identified, I followed that example, and made it a componet(inside list-item) that subscribed its own subscription, which fixed it.

Used the same subscription for updating of `timestamp`